### DISCUSSION TITLE
About the request for help on sot_asr1 in egs2.

### DISCUSSION THREAD

**Discussion by sst0715** [espnet/espnet#5236] [Q&A]
Hey guys, I am trying to reproduce the code of sot_asr1, but it encountered an error as follows: 

<img width="1227" alt="image" src="https://github.com/espnet/espnet/assets/118809740/c19964e2-8386-4c9a-afe3-198fe57fcc3a">

I have checked that the corresponding file exists in the directory and is not damaged, and has all permissions. Could you please advise me on how to solve this problem? Thank you for your help.

**Comment by sst0715** [comment]
路径问题

**Comment by sw005320** [comment]
Is it solved?
It seems that this is simply due to the wrong directory path.

**Reply by sst0715** [reply]
Yeah! I recognized some tips. Thanks for your reply!
